src/components/Input/Input.js

```diff
@@ -20,20 +20,16 @@ export default class Input extends Component {
 
   focus = () => this.input.focus()
 
-  renderErrorText = () => (
-    <Text
-      style={styles.errorText}
-      ellipsizeMode='tail'
-      numberOfLines={1}
-    >
-      {
-        this.props.error || ''
-      }
-    </Text>
-  )
+  handleChange = (value) => {
+    this.props.onChange(this.props.name, value)
```

Comment:
Mb we should use destructuring where we have more than one prop?

Reply:
But here we have only one arg `value`...

Reply:
I mean props and state.
Like here could be:
const {onChange, name} = this.props 
onChange(name, value) for more readability

Reply:
Oh.. In this line... Ok. Done